templates/go/validate_field.tpl

```diff
@@ -56,7 +66,7 @@ if err := support.ValidateFormatUUIDv4(string({{$name}})); err != nil {
     ers = append(ers, err)
 }
 {{- end -}}
-{{- if and $.Object.Format (eq (printf $.Object.Format) "decimal") }}
+{{- if and $.Object.Format (eq (printf $.Object.Format) "decimal") (eq $.Params.decimaltype "string")}}}
```

Comment:
Is `string` a thing that gets set? I thought the default would be empty maybe?

Reply:
We should either change the default value to string, or check if this is empty instead? Not sure which is true, it *looks* like it's just empty.